yrs/src/block.rs

```diff
@@ -523,114 +527,93 @@ impl PartialEq for BlockPtr {
     }
 }
 
-/// An enum containing all supported block variants.
-#[derive(PartialEq)]
-pub(crate) enum Block {
-    /// An active block containing user data.
-    Item(Item),
-
-    /// Block, which is a subject of garbage collection after an [Item] has been deleted and its
-    /// safe for the transaction to remove it.
-    GC(BlockRange),
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub(crate) struct BlockSlice {
```

Comment:
Block slice represents a range of values within a single Block, without need of slicing that block.